### ISSUE TITLE
Setup.py not allowing module import

### ISSUE THREAD

**Issue by deniederhut** [dlab-projects/python-taq#15]
After `python setup.py install`, I'm attempting to import raw_taq from taq and getting this error:

```
Traceback (most recent call last):
  File "taq/generate_test_data.py", line 6, in <module>
    from pytaq import raw_taq
ImportError: No module named 'pytaq'
Majora:python-taq dillon$ python taq/generate_test_data.py /Users/dillon/transfer/EQY_US_ALL_BBO_20140206.zip test_data_public.zip 10
Traceback (most recent call last):
  File "taq/generate_test_data.py", line 6, in <module>
    from taq import raw_taq
ImportError: No module named 'taq'
```

**Comment by rdhyee** [comment]
See whether my PR (https://github.com/dlab-projects/python-taq/pull/14) fixes the problem.

**Comment by deniederhut** [comment]
Closed by 5c3f39a5c4a648754927e55a604bede85dd15a62
